spec/reek/code_comment_spec.rb

```diff
@@ -1,7 +1,6 @@
 require_relative '../spec_helper'
 require_lib 'reek/code_comment'
-require_lib 'reek/errors/bad_detector_configuration_in_comment_error'
-require_lib 'reek/errors/bad_detector_in_comment_error'
+require_lib 'reek/errors'
```

Comment:
I don't think this require are needed, since `code_comment` requires the errors already.